### DISCUSSION TITLE
Pin Multiplexing | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#919] [Q&A]
# Pin Multiplexing | Seeed Studio Wiki

Getting Started with Seeed Studio XIAO ESP32S3.

https://wiki.seeedstudio.com/xiao_esp32s3_pin_multiplexing/

**Comment by fjmsouza** [comment]
Hi, I'm facing problems to read analog values in A10, A9 and A8, using capacitve soil moisture sensor with XIAO expansion board attached. 
obs.1: When change to A1, performs normal reading.
obs.:2: When applied resistive sensor performs normal reading in A10, A9 and A8 pins.

**Comment by reivaxy** [comment]
Hi, unless I'm mistaken your "Digital as PWM" is actually just analog output, and not PWM ?

**Reply by reivaxy** [reply]
Actually I have checked, and the examples does output a square signal whose width varies. But then, no true analog output ?? I've tried modifying the pin_mode, to no avail ;(
